src/test/java/org/springframework/data/hazelcast/repository/PagingSortingIT.java

```diff
@@ -72,7 +72,15 @@ public void unpaged() {
         assertThat("First page is returned", page.getNumber(), equalTo(0));
         assertThat("First page count matches content", page.getNumberOfElements(), equalTo(content.size()));
         assertThat("First page has all content", new Long(page.getNumberOfElements()), equalTo(page.getTotalElements()));
-        assertThat("First page has no upper limit", page.getSize(), equalTo(0));
+        // implementation changed from
```

Comment:
This information is more suitable for a comment in a Pull Request than for a dead commented code in the repository

Reply:
done